### ISSUE TITLE
Allow server-side-only plugins

### ISSUE THREAD

**Issue by calclavia** [NOVA-Team/NOVA-Core#149]
This will allow people to write server-side stuff if they want to. Right now I haven't tested if NOVA (for MC) works without a client installed.

**Comment by asiekierka** [comment]
We need to hook the ModContainer to catch FML's "can connect to other server?" message and bundle extra information about which NOVA mods are installed.

	@NetworkCheckHandler
	public boolean checkNetwork(Map<String,String> mods, Side side) {
		return true;
	}

**Comment by RX14** [comment]
Hmm, I think we shouldn't spend much time in this area, trying to compete with sponge is a mistake.

**Comment by asiekierka** [comment]
@RX14 - We're not going to be competing with Sponge. We should merely have the ability to have client-side-only as well as server-side-only mods. They have a much better "server plugin"-geared API and we know this.

**Comment by RX14** [comment]
I agree with that, however spending time on server-side APIs is a mistake in the near future.

**Comment by Caellian** [comment]
I think you should focus on client-side and both sides until v0.2.0 and focus on this when the basic framework is complete.

**Comment by gjgfuj** [comment]
We still need the ability. At the very least, to support something like
NEI, which doesn't require clients or the server to have it installed, but
will provide additional features if it is.

**Comment by Victorious3** [comment]
It is very easy in fact. `@Mod(acceptableRemoteVersions = "*")`. Just add a flag to our annotation.

**Comment by asiekierka** [comment]
@Victorious3 - I'd say that we should, at least for a later release, encode the mods installed and their versions in the NOVA version marker somehow.

**Comment by RX14** [comment]
We do need to be able to have server side only, and client side only mods. Just don't focus on hooks for it.

(and when I say "don't focus" I *clearly* mean never)

**Comment by RX14** [comment]
I think that we are actually all agreeing here.
